src/cmdlib.py

```diff
@@ -50,3 +53,82 @@ def rm_allow_noent(path):
         os.unlink(path)
     except FileNotFoundError:
         pass
+
+
+def run_supermin(workdir, script=''):
+    vmpreparedir = f"{workdir}/tmp/supermin.prepare"
+    if os.path.isdir(vmpreparedir):
+        shutil.rmtree(vmpreparedir)
+    os.makedirs(vmpreparedir, exist_ok=True)
+
+    vmbuilddir = f"{workdir}/tmp/supermin.build"
+    if os.path.isdir(vmbuilddir):
+        shutil.rmtree(vmbuilddir)
+    os.makedirs(vmbuilddir, exist_ok=True)
+
+    rpms = []
+    with open(f"{libdir}/vmdeps.txt") as vmdeps:
+        for line in vmdeps:
+            if not line or line.startswith('#'):
```

Comment:
I'd write `if line == ""` instead of `if not line` since I have been bitten in the past trying to remember how different languages coerce to booleans.